go.mod

```diff
@@ -581,7 +581,7 @@ replace (
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20190930215403-16217165b5de
 	golang.org/x/mobile => golang.org/x/mobile v0.0.0-20201217150744-e6ae53a27f4f
 	golang.org/x/mod => golang.org/x/mod v0.8.0
-	golang.org/x/net => github.com/openshift-priv/golang-net v0.0.0-20240328080036-60d0f00ca866
```

Comment:
I manually removed this, but seems like one of the vendoring scripts added it back but replaced golang.org/x/net with golang.org/x/net... do we care?